RELEASE.md

```diff
@@ -14,6 +14,7 @@ Release cadence of first pre-releases being cut is 6 weeks.
 | v0.44   | 2020-12-02                                 | Pawel Krupa (GitHub: @paulfantom)           |
 | v0.45   | 2021-01-13                                 | Lili Cosic (GitHub: @lilic)                 |
 | v0.46   | 2021-02-24                                 | **searching for volunteer**                 |
```

Comment:
We are looking for someone to cut the next one 😸

Reply:
if no one volounteers, I can do it ;)

Reply:
Will leave it as searching for now and lets see if anyone volunteers. Otherwise you can add yourself next time :)